lib/recording/RecordingEntrySession.js

```diff
@@ -484,6 +484,24 @@ class RecordingEntrySession {
         this.logger = loggerBase.getLogger('RecordingEntrySession', '[' + this.recordingSessionId + '] ');
         return this.playlistGenerator.setBasePath(this.recordingSessionPath);
     }
+
+    is24By7RecordingEnabled() {
+        return backendClient.getEntryInfo(this.entryObject.entryId)
+            .then((entryObj) => {
+                if (entryObj.capabilities && entryObj.capabilities.indexOf('FEATURE_KALTURA_24_BY_7_RECORDING') >= 0) {
+                    this.logger.info(`Feature 24 by 7 is recording is enabled. Recording will be split`);
```

Comment:
sentence doesn't make sense.
If you changed the feature name, then rephrase the sentence as well:
"Feature Continuous recording is enabled..."